### ISSUE TITLE
type-punned pointers

### ISSUE THREAD

**Issue by rofl0r** [kometbomb/klystron#16]

**Comment by kometbomb** [comment]
Yeah I need to check if that's actually needed anymore. The original code relied on being able to pass objects to the library functions that were not exactly ObjHdr etc. but had that as the "headed" of the data.
